src/mainwindow.cpp

```diff
@@ -3841,6 +3841,8 @@ void MainWindow::setCurrentNote(Note note, bool updateNoteText,
 
     // clear external image cache
     Note::externalImageHash()->clear();
+
+    ui->actionToggle_distraction_free_mode->setEnabled(true);
```

Comment:
Why do you want to turn on the distraction free mode every time the note changes (even if not in distraction free mode)?

Reply:
Ah, sorry. you enabled the checkbox!